radix-common/src/constants/transaction_validation.rs

```diff
@@ -20,6 +11,3 @@ pub const MAX_TIP_PERCENTAGE: u16 = u16::MAX;
 /// The max epoch range
 /// Should be ~ 1 month. The below is ~30 days given 5 minute epochs.
 pub const MAX_EPOCH_RANGE: u64 = 12 * 24 * 30;
```

Comment:
why don't we move `MIN/MAX_TIP_PERCENTAGE` and `MAX_EPOCH_RANGE`  to `PreparationSettings` too?

Reply:
Good question... I guess conceptually I want preparation to be simple limits (for the sake of building a valid payload hash, for things that a low-level client such as a ledger might need to worry about) - whereas the validation is for any non-trivial logic, and applied specifically to user transactions (not test transactions).

I'll add some comments in a later PR to explain the difference :D